components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilderTest.java

```diff
@@ -199,7 +203,7 @@ public void testAddFragmentParameter() {
     public void testAddFragmentParameters() {
 
         String fragment = null;
-
+        when(CarbonUtils.getManagementTransport()).thenReturn("https");
```

Comment:
get "https" to a constant

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3111/commits/beb53f7154b1bbf61aeb6b1130af4fc849b4b681